pkg/store/cache/caching_bucket_factory.go

```diff
@@ -30,6 +38,41 @@ type CachingBucketWithBackendConfig struct {
 	CachingBucketConfig CachingBucketConfig `yaml:"caching_config"`
 }
 
+type CachingBucketConfig struct {
+	// Basic unit used to cache chunks.
+	ChunkSubrangeSize int64 `yaml:"chunk_subrange_size"`
+
+	// Maximum number of GetRange requests issued by this bucket for single GetRange call. Zero or negative value = unlimited.
+	MaxChunksGetRangeRequests int `yaml:"max_chunks_get_range_requests"`
+
+	// TTLs for various cache items.
+	ChunkObjectSizeTTL time.Duration `yaml:"chunk_object_size_ttl"`
+	ChunkSubrangeTTL   time.Duration `yaml:"chunk_subrange_ttl"`
+
+	// How long to cache result of Iter call in root directory.
+	RootIterTTL time.Duration `yaml:"root_iter_ttl"`
+
+	// Config for Exists and Get operations for metadata files.
+	MetafileExistsTTL      time.Duration `yaml:"metafile_exists_ttl"`
+	MetafileDoesntExistTTL time.Duration `yaml:"metafile_doesnt_exist_ttl"`
+	MetafileContentTTL     time.Duration `yaml:"metafile_content_ttl"`
+}
+
+func DefaultCachingBucketConfig() CachingBucketConfig {
+	return CachingBucketConfig{
+		ChunkSubrangeSize:         16000, // Equal to max chunk size.
+		ChunkObjectSizeTTL:        24 * time.Hour,
+		ChunkSubrangeTTL:          24 * time.Hour,
+		MaxChunksGetRangeRequests: 3,
+
+		RootIterTTL: 5 * time.Minute,
+
+		MetafileExistsTTL:      10 * time.Minute,
+		MetafileDoesntExistTTL: 3 * time.Minute,
+		MetafileContentTTL:     1 * time.Hour,
```

Comment:
Metadata file is expected to be immutable, so I would rase the default to 24h like chunks.

Reply:
This works for now, since `BaseFetcher` uses `Exists` method to verify if meta.json file exists before fetching it. If it switches to `Get` only, we may need to reconsider this default value.